x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -8323,7 +8323,6 @@
     "xpack.fleet.agentList.policyColumnTitle": "代理策略",
     "xpack.fleet.agentList.policyFilterText": "代理策略",
     "xpack.fleet.agentList.reassignActionText": "分配到新策略",
-    "xpack.fleet.agentList.revisionNumber": "修订版 {revNumber}",
```

Comment:
Not fully sure how this works. But it seems the `revisionNumber` is still available in the new code?